include/capstone/capstone.h

```diff
@@ -84,6 +84,7 @@ typedef enum cs_arch {
 	CS_ARCH_TMS320C64X,	///< TMS320C64x architecture
 	CS_ARCH_M680X,		///< 680X architecture
 	CS_ARCH_EVM,		///< Ethereum architecture
+	CS_ARCH_RISCV,          ///< RISCV architecture
```

Comment:
@aquynh I think a new architecture should always be inserted in the last? I mean, `CS_ARCH_RISCV` should be put right before `CS_ARCH_MAX`, otherwise the value of the enum will be different.

Reply:
you are right, that is a good practice. i think the reason is this work started even before WASM & BPF, so when he ported it, he keeps that in the same place.

however, this is not a big deal, because all of the new archs are only in the next branch.

in case RICSV is moved down, keep in mind that RISCV position in arrays cs_arch_init, cs_arch_option, cs_arch_disallowed_mode_mask should be updated, too.

Reply:
yes, the first place I have chosed is after EVM.

Reply:
can you please move RISCV to the end, after BPF? otherwise later we add RISCV to bindings, the order will mess up.

Reply:
this is the reason why that test_mos65xx.py breaks right now in Travis.